### PR TITLE
321 bug fixes

### PR DESCRIPTION
**Pull request by mcphedar**
Fixes the teleport locations that were placed in a tree and adds a few more late game gem spawns to complicate the "balfour speed run"

### CODE REVIEW

File: Levels/NewStarbase/NewStarbase.prefab [hunk 1]
```diff
@@ -4383,8 +4376,9 @@
                     "Id": 8615741586168280576,
                     "m_template": {
                         "$type": "MultiplayerSample::HUDComponent",
-                        "RoundNumberId": 8,
-                        "RoundTimerId": 4
+                        "RoundNumberId": "",
+                        "RoundTimerId": "",
+                        "RoundSecondsRemaining": ""
```
Comment (mbalfour-amzn) [inline Levels/NewStarbase/NewStarbase.prefab:4381]: Is this change intentional? If so, what does it fix? If not, does the HUD still work?

Comment (mcphedar) [reply Levels/NewStarbase/NewStarbase.prefab:4381]: No this was not something I changed and no, the HUD doesn't work. I know it worked in testing earlier, so i'm not sure what happened.
